books/RayTracingInOneWeekend.html

```diff
@@ -466,9 +466,9 @@
     }
 
     int main() {
-        const int image_width = 768;
-        const auto aspect_ratio = 9.0 / 16.0;
-        const int image_height = static_cast<int>(aspect_ratio * image_width);
+        const auto aspect_ratio = 16.0 / 9.0;
+        const int image_width = 384;
```

Comment:
image height * aspect seems more intuitive

Reply:
W/H is consistent with the aspect ratio we've been using for the camera code. Specifying the single image size parameter as width feels more natural as it's the maximum dimension, and text also typically has a sensible bounded and known width (while height is varying).